HW8/OlenaBudko/Task_8_3/utils/__init__.py

```
@@ -0,0 +1,9 @@
+import sys
+import os
+
+sys.path.append(os.getcwd())
+
+from .calculator import *
```

Ruff (0.8.2)

`from .calculator import *` used; unable to detect undefined names

(F403)